flexmeasures/data/services/scheduling.py

```diff
@@ -66,9 +91,13 @@ def create_scheduling_job(
     scheduler = find_scheduler_class(sensor)(sensor=sensor, **scheduler_kwargs)
     scheduler.deserialize_config()
 
+    on_failure_cb = None
+    if (scheduler.fallback_scheduler_class is not None) and not is_fallback:
```

Comment:
I thought we were going for chaining here? That is, dispensing with the `is_fallback` bool that is effectively limiting the chain to two links (i.e. a single fallback step).

Reply:
There was the issue that we couldn't know if the call was due a direct schedule or a failing schedule. I change the logic and now it supports chaining (added a test).